Eastern-Research-Group/mywaterway
# Feature/296 monitoring report changes

## Related Issues:
* https://jira.epa.gov/browse/HMW-296

## Main Changes:
* Changed the route to the monitoring report page to `monitoring-report` from `location`.
* Renamed the monitoring report and associated test files to match the new route name.
* Changed some text on the monitoring report page.
* Updated the monitoring report tests to work with the new route/text.

## Steps To Test:
1. Navigate to http://localhost:3000/community/dc/overview
2. Click `View Monitoring Report` on one of the past water conditions items
3. Verify it navigates to a url with `/monitoring-report` 
4. Navigate to http://localhost:3000/monitoring-report/STORET/CBP_WQX/CBP_WQX-01651770/
5. Verify the page loads
6. Verify the text changes match up with those listed in the ticket
7. Run the `monitoring-report.cy.ts` tests
8. Verify they all pass
